.gitignore

```diff
@@ -62,3 +62,12 @@ out
 
 # Visual Studio Code
 .vscode
+
+### Artifacts ###
+
+# Artifacts
+*.tgz
+target
+dist
+.interfaces
```

Comment:
I think I'd like to see this near the top. Perhaps rename "Log files" to "Artifacts" and move them there?

(And sort them please.)

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6348/commits/ab00a4dcab773ba71746934783f82b1b88669b02